config/locales/en.yml

```diff
@@ -23,3 +23,7 @@ en:
   hello: "Hello world"
   sets:
     set_not_found: "Invalid set requested."
+  hydra_editor:
+    form:
+      model_label:
+        GenericSet: "Collection"
```

Comment:
@wickr You seemed to have the most opinions on this before - should the form for ingest say "Set" or "Collection"?